package.json

```diff
@@ -4,23 +4,117 @@
   "description": "Lambda Function to Stream Logs from AWS S3 to LogDNA",
   "main": "index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint -c .eslintrc *.js"
+    "docs": "config-doc lib/config.js > doc/env.md",
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "tap": "tap",
+    "pretest": "npm run lint",
+    "pretest:ci": "npm run lint",
+    "test": "npm run tap",
+    "test:ci": "tools/test-ci.sh",
+    "package": "tools/package.sh",
+    "release": "semantic-release"
+  },
+  "release": {
+    "branches": [
+      "master"
+    ],
+    "plugins": [
+      [
+        "@semantic-release/github",
+        {
+          "assets": [
+            {
+              "path": "pkg/logdna-s3.zip"
+            }
+          ]
+        }
+      ]
+    ],
+    "extends": "semantic-release-config-logdna"
+  },
+  "eslintConfig": {
+    "root": true,
+    "ignorePatterns": [
+      "node_modules/",
+      "coverage/"
+    ],
+    "extends": [
+      "logdna"
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2019
+    }
+  },
+  "tap": {
+    "esm": false,
+    "ts": false,
+    "jsx": false,
+    "browser": false,
+    "check-coverage": true,
+    "output-file": ".tap",
+    "timeout": 60,
+    "nyc-arg": [
+      "--all=true",
+      "--exclude=test/"
+    ],
+    "coverage-report": [
+      "html",
+      "text",
+      "text-summary",
+      "json",
+      "json-summary"
+    ],
+    "files": [
+      "test/integration",
+      "test/unit"
+    ],
+    "statements": 0,
+    "branches": 0,
+    "functions": 0,
+    "lines": 0
   },
   "dependencies": {
-    "agentkeepalive": "^4.0.2",
-    "async": "^2.6.2",
-    "request": "^2.88.0"
+    "@logdna/env-config": "^1.0.5",
+    "@logdna/logger": "^2.3.0"
   },
   "devDependencies": {
+    "@commitlint/cli": "^9.1.2",
+    "@commitlint/config-conventional": "^12.1.1",
+    "@semantic-release/github": "^7.2.1",
     "aws-sdk": "^2.590.0",
-    "eslint": "^6.7.2"
+    "eslint": "^7.4.0",
+    "eslint-config-logdna": "^4.0.2",
```

Comment:
nit: should be able to bump this to 5.x, and tap to 15.x as part of this, and `npm run lint:fix`. not necessary for this PR but would be a quick change in an additional commit if you wanted to include it.

Reply:
Updated their versions